verifier.py

```diff
@@ -355,31 +355,68 @@ def __init__(self):
     def __init__(self):
         Verifier.__init__(self)
         self.pages = []
         self.minimum_size = 350
         self.page_elements = [
             '#fileBrowser > div.db-infobar > div > div',  # Project preview
-            '#tb-tbody'                                   # Project browser
+            '#tb-tbody'  # Project browser
         ]
         self.harvest_pages(run_info['institution_urls'], '', InstitutionProfilePage)
         self.size_comparison()
         self.spot_check()
 
 
-# Main execution
-
-RegistrationFiles = RegistrationFilesVerifier()
-
-print("Retry list: ", RegistrationFiles.failed_pages)
+def main():
+    # for modularization and then concatenate lists
+    rescrape_list = []
+    for i in range(NUM_RETRIES):
+        if run_info['scrape_nodes']:
+            if run_info['include_files']:
+                project_files = ProjectFilesVerifier()
+                rescrape_list.append(project_files.failed_pages)
+            if run_info['include_wiki']:
+                project_wiki = ProjectWikiVerifier()
+                rescrape_list.append(project_wiki.failed_pages)
+            if run_info['include_analytics']:
+                project_analytics = ProjectAnalyticsVerifier()
+                rescrape_list.append(project_analytics.failed_pages)
+            if run_info['include_registrations']:
+                project_registrations = ProjectRegistrationsVerifier()
+                rescrape_list.append(project_registrations.failed_pages)
+            if run_info['include_forks']:
+                project_forks = ProjectForksVerifier()
+                rescrape_list.append(project_forks.failed_pages)
+            if run_info['include_dashboard']:  # This must go last because its URLs don't have a specific ending.
+                project_dashboards = ProjectDashboardVerifier()
+                rescrape_list.append(project_dashboards.failed_pages)
+        if run_info['scrape_registrations']:
+            # Must run all page types automatically
+            registration_files = RegistrationFilesVerifier()
+            registration_wiki = RegistrationWikiVerifier()
+            registration_analytics = RegistrationAnalyticsVerifier()
+            registration_forks = RegistrationForksVerifier()
+            registration_dashboards = RegistrationDashboardVerifier()
+            rescrape_list.extend((registration_files.failed_pages, registration_wiki.failed_pages,
+                                  registration_analytics.failed_pages, registration_forks.failed_pages,
+                                  registration_dashboards.failed_pages))
+        if run_info['scrape_users']:
+            user_profiles = UserProfileVerifier()
+            rescrape_list.append(user_profiles.failed_pages)
+        if run_info['scrape_institutions']:
+            institution_dashboards = InstitutionDashboardVerifier()
+            rescrape_list.append(institution_dashboards.failed_pages)
+
+
+if __name__ == "__main__": main()
```

Comment:
I left that there for cases of unit testing. Then we would want to so we can test specific methods from this file.